src/bin/ion/commands/beta/generate/generator.rs

```diff
@@ -124,9 +144,9 @@ impl<'a, L: Language> CodeGenerator<'a, L> {
         _map: &HashMap<String, tera::Value>,
     ) -> Result<tera::Value, tera::Error> {
         Ok(tera::Value::Bool(L::is_built_in_type(
-            value
-                .as_str()
-                .ok_or(tera::Error::msg("Required string for this filter"))?,
+            value.as_str().ok_or(tera::Error::msg(
```

Comment:
Same thought here RE: taking a `&str`.

Reply:
This [as_str](https://docs.rs/tera/1.19.1/tera/enum.Value.html#method.as_str) here is a method on tera::Value. It converts a tera value as string similar to Element::as_str(), which is why I have used as_str on these filters (snake or upper_camel).